frontend/app/routes/$lang+/_public+/apply+/$id+/personal-information.tsx

```diff
@@ -151,26 +173,47 @@ export async function action({ request, params }: ActionFunctionArgs) {
 
         if (!isEmpty(val.homeCountry)) {
           if ((val.homeCountry === CANADA_COUNTRY_ID || val.homeCountry === USA_COUNTRY_ID) && isEmpty(val.homeProvince)) {
-            ctx.addIssue({ code: z.ZodIssueCode.custom, message: 'empty-province', path: ['homeProvince'] });
+            ctx.addIssue({ code: z.ZodIssueCode.custom, message: t('apply:personal-information.error-message.empty-province'), path: ['homeProvince'] });
           }
           if ((val.homeCountry === CANADA_COUNTRY_ID || val.homeCountry === USA_COUNTRY_ID) && isEmpty(val.homePostalCode)) {
-            ctx.addIssue({ code: z.ZodIssueCode.custom, message: 'empty-postal-code', path: ['homePostalCode'] });
+            ctx.addIssue({ code: z.ZodIssueCode.custom, message: t('apply:personal-information.error-message.empty-postal-code'), path: ['homePostalCode'] });
           }
         }
 
         if (!isValidPostalCode(val.homeCountry, val.homePostalCode)) {
-          ctx.addIssue({ code: z.ZodIssueCode.custom, message: val.homeCountry === CANADA_COUNTRY_ID ? 'invalid-postal-code' : 'invalid-zip-code', path: ['homePostalCode'] });
+          ctx.addIssue({
+            code: z.ZodIssueCode.custom,
+            message: val.homeCountry === CANADA_COUNTRY_ID ? t('apply:personal-information.error-message.invalid-postal-code') : t('apply:personal-information.error-message.invalid-zip-code'),
+            path: ['homePostalCode'],
+          });
         }
       }
     });
 
-  const formData = Object.fromEntries(await request.formData());
+  const formData = await request.formData();
+  const personalInformation = {
```

Comment:
Only set fields as `String` if they are not required. Some fields like phoneNumber and phoneNumberAlt are optional.

```ts
phoneNumber: formData.get('phoneNumber') && String(formData.get('phoneNumber')),
phoneNumberAlt: formData.get('phoneNumberAlt') && String(formData.get('phoneNumberAlt')),
```